specs/protocol/jovian/calldata-block-limit.md

```diff
@@ -0,0 +1,51 @@
+
+# Calldata Limit
+Starting with Jovian, the `gasUsed` field of a block may instead represent the utilization of calldata within the block.
+The purpose of this feature is to take into account the throughput of the DA Layer (the L1) when creating blocks that feature a high volume of `calldata`
+
+# Terms 
+
+## `calldata`
+Calldata is the `data` field attached to a transaction submitted to the OP Chain. The transaction author fully controls the content and size of the calldata at the time of authoring.
+
+## `da_bytes_estimate`
+`da_bytes_estimate` is the size of the `calldata` present in a transaction with respect to how much space it is expected to consume on the DA Layer. The DA Bytes Estimate may be smaller thatn the raw Calldata because the data is compressed when published to L1.
+
+```py
+da_bytes_estimate = max(min_transaction_size, intercept + fastlz_coef * tx.fastlz_size / 1e6)
+```
+
+The protocol constants `min_transaction_size`, `intercept`, `fastlz_coef` are defined in prior Hard Forks.
```

Comment:
Can link to Fjord here https://specs.optimism.io/protocol/fjord/exec-engine.html#fjord-l1-cost-fee-changes-fastlz-estimator